README.md

```diff
@@ -47,56 +47,56 @@ _Note_: When you create a Git commit, any staged scripts will be automatically r
 
 These scripts can all be run from the root level of the repo:
 
-* `yarn start`
-  * Starts local server running the documentation site
-  * Regenerates documentation when files change
-* `yarn build`
-  * Compile/transpile/uglify everything and makes things release-ready.
-* `yarn bump`
-  * Increments package versions. Read "[Versioning](/guides/RELEASE-PROCESS.md#versioning)" for more info.
-* `yarn generate`
-  * Generates the necessary files for a new core component
-  * Alias: `yarn g`
-* `yarn test`
-  * Runs JS unit tests
-  * Lints JS using ESLint
-  * Lints Sass using stylelint
-* `yarn test:e2e`
-  * Runs end to end tests
-* `yarn test:e2e packages/core/Autocomplete`
-  * Runs a single end to end test, this example runs Autocomplete
-* `yarn test:watch`
-  * Runs JS unit tests and will continue to run tests as files change
-* `yarn update-snapshots`
-  * Updates [Jest snapshots](http://facebook.github.io/jest/docs/en/snapshot-testing.html)
-* `yarn lint`
-  * Runs just the linting portion of the tests
+- `yarn start`
+  - Starts local server running the documentation site
+  - Regenerates documentation when files change
+- `yarn build`
+  - Compile/transpile/uglify everything and makes things release-ready.
+- `yarn bump`
+  - Increments package versions. Read "[Versioning](/guides/RELEASE-PROCESS.md#versioning)" for more info.
+- `yarn generate`
+  - Generates the necessary files for a new core component
+  - Alias: `yarn g`
+- `yarn test`
+  - Runs JS unit tests
+  - Lints JS using ESLint
+  - Lints Sass using stylelint
+- `yarn test:e2e`
+  - Runs end to end tests
+- `yarn test:e2e packages/core/Autocomplete`
+  - Runs a single end to end test, this example runs Autocomplete
+- `yarn test:watch`
+  - Runs JS unit tests and will continue to run tests as files change
+- `yarn update-snapshots`
+  - Updates [Jest snapshots](http://facebook.github.io/jest/docs/en/snapshot-testing.html)
+- `yarn lint`
+  - Runs just the linting portion of the tests
```

Comment:
There are a few images that are broken 😢 

- backstop_default_CMSDS_-_Components_-_Select_0_document_1_tablet.png
- backstop_default_CMSDS_-_Utilities_-_Clearfix_0_document_0_phone.png
- backstop_default_CMSDS_-_Utilities_-_Clearfix_0_document_1_tablet
- backstop_default_CMSDS_-_Utilities_-_Clearfix_0_document_2_desktop
- backstop_default_CMSDS_-_Utilities_-_Truncate_0_document_0_phone
- backstop_default_CMSDS_-_Utilities_-_Truncate_0_document_1_tablet
- backstop_default_CMSDS_-_Utilities_-_Truncate_0_document_2_desktop